keyserver/README.md

```diff
@@ -0,0 +1,30 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+Generate new config file with proper format
+```
+go run utils/configen/mkconfig.go
+```
+
+Generate new key pair for signing
+``` 
+go run utils/keygen/mkkey.go -vrf -signing
+```
+
+Generate a private key for secure connection (TLS)
+```
+openssl ecparam -genkey -name prime256v1 -out server.key
+```
+
+Generation of self-signed(x509) public key (PEM-encodings `.pem`) based on the private (`.key`)
+```
+openssl req -new -x509 -sha256 -key server.key -out server.pem -days 3650
+```
+
+Run the server
+```
+go run keyserver/coniksserver/main.go
+```
```

Comment:
Is this supposed to be a complete set of instructions? It seems like the signing key needs to be generated for this to work and the above command `go run utils/keygen/mkkey.go -vrf -signing` only uses the first flag. Hence, trying to run the server will fail.